src/Conference/ConferenceDetails.jsx

```diff
@@ -56,7 +55,7 @@ const ConferenceDetails = ({ conference, id, isLoading, error }) => {
         <StyledDescription className="description">
           {description}
         </StyledDescription>
-        <Button onClick={deleteThisConference}>Delete Conference</Button>
+        <button onClick={deleteThisConference}>Delete Conference</button>
```

Comment:
not a fan of your own button?

Reply:
Our `Button` didn't really have any styling on it and we wanted to add styling to the logout button to make it fit in with the rest of the nav bar, so we decided it was easier to use the default button for the delete button and then style the custom one instead of adding functionality to change the styling of the `Button` by passing props. We could definitely add that functionality in if that'd be a better solution?

Reply:
If the reusable button is not reusable, maybe it's time to re-think how it's been implemented! Reusable components should be able to have their styling adapted (For example we pass in a classname and use it to override the css)

Reply:
Latest commit adds this functionality, much easier than we thought it would be! 😄